Add clarification about OpenSearch ingest pipelines compared to Data … (#6786)
opensearch-project/documentation-website: _ingest-pipelines/index.md

Processors are customizable tasks that run in a sequential order as they appear in the request body. This order is important, as each processor depends on the output of the previous processor. The modified documents appear in your index after the processors are applied.

## OpenSearch ingest pipelines compared to Data Prepper

> OpenSeach ingest pipelines run within the OpenSearch cluster, whereas [Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/) is an external component that runs off the OpenSearch cluster. 

Reviewer: ```suggestion
OpenSeach ingest pipelines run within the OpenSearch cluster, whereas [Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/) is an external component that runs on the OpenSearch cluster. 
```

> OpenSearch ingest pipelines perform actions on indexes and are preferred for use cases involving pre-processing simple datasets, [machine learning processors]({{site.url}}{{site.baseurl}}/ingest-pipelines/processors/sparse-encoding/), and [vector embedding processors]({{site.url}}{{site.baseurl}}/ingest-pipelines/processors/text-image-embedding/). OpenSearch ingest pipelines are recommended for simple data pre-processing and small datasets. 

Reviewer: ```suggestion
OpenSearch ingest pipelines perform actions on indexes and are preferred for use cases involving pre-processing simple datasets, [machine learning (ML) processors]({{site.url}}{{site.baseurl}}/ingest-pipelines/processors/sparse-encoding/), and [vector embedding processors]({{site.url}}{{site.baseurl}}/ingest-pipelines/processors/text-image-embedding/). OpenSearch ingest pipelines are recommended for simple data pre-processing and small datasets. 
```

> Data Prepper is the preferred data ingestion tool for OpenSearch. Data Prepper is recommended for any data processing that it supports and for use cases involving the transferring and fetching of large datasets and complex data pre-processing. Refer to the [Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/) documentation to learn more information.      

Reviewer: ```suggestion
Data Prepper is the preferred data ingestion tool for OpenSearch and is recommended for any data processing that it supports and for use cases involving the transfer and fetching of large datasets and complex data pre-processing. Refer to the [Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/) documentation for more information.      
```

> Data Prepper is the preferred data ingestion tool for OpenSearch. Data Prepper is recommended for any data processing that it supports and for use cases involving the transferring and fetching of large datasets and complex data pre-processing. Refer to the [Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/) documentation to learn more information.      

Reviewer: First sentence: This reads as though a noun should follow "data processing". Data processing what that it supports?

Author: Is this a better rewrite? `Data Prepper is recommended for any data processing tasks it supports, particularly when dealing with large datasets and complex data pre-processing requirements. It streamlines the process of transferring and fetching large datasets, while providing robust capabilities for intricate data preparation and transformation operations.`

Reviewer: Yes, that works, but no comma in the second sentence 😄

> OpenSearch ingest pipelines can only be managed using [ingest API operations]({{site.url}}{{site.baseurl}}/api-reference/ingest-apis/index/).

Reviewer: In the link, should "Ingest" be capitalized (is it the name of the API)?
